src/module-1/card/card.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="card">
+        <div class="card__wrapper">
+            <div class="card__image-container">
+                <img src="apple-watch.svg" alt="" class="card__image">
+            </div>
+            <div class="card__rating">
+                <div class="card-rating__stars">
+                    <div class="card-rating__stars-container">
+                        3.4
+                    </div>
+                    <img src="star.svg" alt="" class="stars__star">
+                </div>
+                <div class="card-rating__price">$399</div>
+            </div>
+            <div class="card__description">
+                <div class="card-description__title">Apple Watch Series 4 GPS</div>
+                <div class="card-description__desc">Redesigned from scratch and completely revised.</div>
+            </div>
+        </div>
+        <div class="card__bottom">
+            <div class="card-bottom__wishlist"> <img src="wishlist.svg" alt=""> WISHLIST</div>
```

Comment:
uppercase may be achieved with css `text-transform` as well, so you don't need to put text in caps